apps/trustlab/src/payload/access/anyone.js

```diff
@@ -0,0 +1,3 @@
+/* eslint-disable import/prefer-default-export */
+
+export const anyone = () => true;
```

Comment:
An alternative to disabling this is:
```suggestion
export const anyone = () => true;

export default undefined;
```